### ISSUE TITLE
Exception in Nodes constructor (firmware 5.0.10)

### ISSUE THREAD

**Issue by hoopty** [automicus/PyISY#28]
I'm getting the following exception on 5.0.10.  It started after I added a Fortrezz Mimo Lite device to the controller, it was working before that with locks, switches, and lamps.

```bash
Traceback (most recent call last):
  File "/home/hass/hass/lib/python3.6/site-packages/homeassistant/setup.py", line 190, in _async_setup_component
    component.setup, hass, processed_config)
  File "/usr/local/lib/python3.6/asyncio/futures.py", line 331, in __iter__
    yield self  # This tells Task to wait for completion.
  File "/usr/local/lib/python3.6/asyncio/tasks.py", line 244, in _wakeup
    future.result()
  File "/usr/local/lib/python3.6/asyncio/futures.py", line 244, in result
    raise self._exception
  File "/usr/local/lib/python3.6/concurrent/futures/thread.py", line 55, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/hass/hass/lib/python3.6/site-packages/homeassistant/components/isy994.py", line 192, in setup
    use_https=https, tls_ver=tls_version, log=_LOGGER)
  File "/home/hass/.homeassistant/deps/PyISY/ISY.py", line 84, in __init__
    self.nodes = Nodes(self, xml=self.conn.getNodes())
  File "/home/hass/.homeassistant/deps/PyISY/Nodes/__init__.py", line 73, in __init__
    self.parse(xml)
  File "/home/hass/.homeassistant/deps/PyISY/Nodes/__init__.py", line 220, in parse
    nval = int(nval.replace(' ', '0'))
ValueError: invalid literal for int() with base 10: ''
```

**Comment by bing281** [comment]
Can we merge this into a new version?

**Comment by rmkraus** [comment]
This should be resolved in version 1.0.8
